Source/DafnyLanguageServer/Language/Symbols/SymbolTable.cs

```diff
@@ -62,7 +67,17 @@ bool symbolsResolved
     /// <returns><c>true</c> if a symbol was found, otherwise <c>false</c>.</returns>
     /// <exception cref="System.InvalidOperationException">Thrown if there was one more symbol at the specified position. This should never happen, unless there was an error.</exception>
     public bool TryGetSymbolAt(Position position, [NotNullWhen(true)] out ILocalizableSymbol? symbol) {
-      symbol = LookupTree.Query(position).SingleOrDefault();
+      var symbolsAtPosition = LookupTree.Query(position);
+      symbol = null;
+      // Use case: function f(a: int) {}, and hover over a.
+      foreach (var potentialSymbol in symbolsAtPosition) {
+        if (symbol != null) {
+          logger.Log(LogLevel.Warning, $"Two registered symbols as the same position (line {position.Line}, character {position.Character})");
```

Comment:
`ILogger` do not fully support interpolated strings (yet). Use a log-format with arguments:
```cs
logger.Log(LogLevel.Warning, "Two registered symbols as the same position (line {Line}, character {Character})", position.Line, position.Character);
```

Reply:
Normally if there is no interpolation handler, I thought "$" would just do string concatenation.
But I modify it according to your snippet.

Reply:
You're right. However, the recent Log4Shell hustle makes it more evident than ever to use the logging frameworks carefully (besides negligible performance considerations).
I noted the string interpolation handler since I suppose that will be a feature of various logging frameworks eventually.